Controller/Payment/Callback.php

```diff
@@ -57,14 +62,16 @@ public function __construct(
         \Magento\Sales\Model\Service\InvoiceService $invoiceService,
         \Magento\Sales\Model\Order\Email\Sender\InvoiceSender $invoiceSender,
         \Magento\Framework\DB\Transaction $transaction,
-        \Kevin\Payment\Logger\Logger $logger
+        \Kevin\Payment\Logger\Logger $logger,
+        \Kevin\Payment\Helper\Data $kevinHelper
```

Comment:
Replace qualifiers with imports (use `use` statements instead of full namespaces every time) all throughout. PHP <5.3 support is irrelevant.

Reply:
@tomas-maciulis let's leave it as it is and stick to the code conventions in the module